storage/innobase/xtrabackup/doc/Makefile

```diff
@@ -155,14 +155,3 @@ doctest:
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
 
-thtml:
-	@echo "Building themed html doc"
-	$(SPHINXBUILD) -D html_theme=alabaster -D html_theme_path='' -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
-	@echo
-	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
-
-livehtml:
-	sphinx-autobuild $(ALLSPHINXOPTS) $(BUILDDIR) $(O)
-
-livethtml:
-	sphinx-autobuild -D html_theme=alabaster -D html_theme_path='' $(ALLSPHINXOPTS) $(BUILDDIR) $(O)
```

Comment:
what about removing unused files? all those in builddir/html are no longer required?